lib/mix/lib/mix/dep.ex

```diff
@@ -209,16 +209,23 @@ defmodule Mix.Dep do
     do: "the dependency does not match the requirement #{inspect req}, got #{inspect vsn}"
 
   def format_status(%Mix.Dep{status: {:lockmismatch, _}}),
-    do: "lock mismatch: the dependency is out of date (run \"mix deps.get\" to fetch locked version)"
```

Comment:
Let's go with:

> To fetch locked version run "mix deps.get"

Let's remove the `:` from all of those as it only works if the command is the last thing on the sentence.